core/src/main/java/org/opensearch/sql/expression/function/FunctionDSL.java

```diff
@@ -183,6 +183,60 @@ public String toString() {
     };
   }
 
+  /**
+   * Implementation of a function that takes three arguments, returns a value, and
+   * requires FunctionProperties to complete.
+   *
+   * @param function   {@link ExprValue} based Binary function.
+   * @param returnType return type.
+   * @param args1Type first argument type.
+   * @param args2Type second argument type.
+   * @param args3Type second argument type.
```

Comment:
```suggestion
   * @param args3Type third argument type.
```

Reply:
Changed in https://github.com/Bit-Quill/opensearch-project-sql/commit/77226baed22abe00079b735df28ff26f2cc7547e